README.md

```diff
@@ -651,6 +651,7 @@ Latest ✔ | Latest * ✔ | 11 ✔ | Latest ✔ | Latest ✔ |
 
 ### Troubleshooting
 
+#### CSP issues
```

Comment:
I think using the full name is better/clearer for the heading here

Reply:
I agree. Done